xmtp_mls/src/utils/address.rs

```diff
@@ -0,0 +1,28 @@
+use thiserror::Error;
+use xmtp_cryptography::signature::is_valid_ethereum_address;
+
+#[derive(Debug, Error)]
+pub enum AddressValidationError {
+    #[error("invalid addresses: {0:?}")]
+    InvalidAddresses(Vec<String>),
+}
+
+pub fn sanitize_evm_addresses(
```

Comment:
Moved this out of `groups/mod.rs` and made it lowercase the addresses after validation